services/pgnotifier/pgnotifier.go

```diff
@@ -355,6 +365,31 @@ func (notifier *PgNotifierT) Publish(jobs []JobPayload, priority int) (ch chan [
 		pkgLogger.Errorf("PgNotifier: Error publishing messages: %v", err)
 		return
 	}
+
+	uploadSchemaJSON, err := json.Marshal(struct {
+		UploadSchema whUtils.SchemaT
+	}{
+		UploadSchema: *schema,
+	})
+	if err != nil {
+		return
+	}
+
+	sqlStatement := `
+		UPDATE
+		  pg_notifier_queue
+		SET
+		  payload = payload || $1
+		where
+		  batch_id = $2;`
+	_, err = txn.Exec(sqlStatement, []interface{}{
+		uploadSchemaJSON,
+		batchID,
+	}...)
+	if err != nil {
+		return
```

Comment:
When returning, it's always good to add some context, so maybe:

`return fmt.Errorf("Unable to execute transaction to update notifier queue, err: %w", err)`

Reply:
Done.